### PR TITLE
Add autofix to media-feature-parentheses-space-inside

### PR DESCRIPTION
**Pull request by yepninja**
#3585

### CODE REVIEW

**Comment by yepninja** [comment]
I had to fix snapshots. Now the order of the errors is more correct.

File: lib/rules/media-feature-parentheses-space-inside/index.js [hunk 1]
```diff
@@ -31,50 +31,62 @@ const rule = function(expectation) {
       // will be at atRule.raws.params.raw
       const params = _.get(atRule, "raws.params.raw", atRule.params);
       const indexBoost = atRuleParamIndex(atRule);
+      const errors = [];
```
Comment (jeddy3) [inline lib/rules/media-feature-parentheses-space-inside/index.js]: Can we rename this to `violations`, please?

It's a convention we've adopted to:

- help distinguish between things caught by the linter and actual errors, for example thrown by the parsers.
- act as an intermediary before those things are categorised into a particular severity e.g. "error" or "warning".

Comment (yepninja) [reply lib/rules/media-feature-parentheses-space-inside/index.js]: Agreed. No problem at all.

Comment (yepninja) [reply lib/rules/media-feature-parentheses-space-inside/index.js]: Done!

File: lib/rules/media-feature-parentheses-space-inside/index.js [hunk 2]
```diff
@@ -31,50 +31,62 @@ const rule = function(expectation) {
...
+            }
+            if (node.after === " ") {
+              if (context.fix) node.after = "";
+              errors.push({
+                message: messages.rejectedClosing,
+                index: node.sourceIndex - 2 + len + indexBoost
+              });
+            }
+          }
+          if (expectation === "always") {
```
Comment (vankop) [inline lib/rules/media-feature-parentheses-space-inside/index.js]:
```suggestion
         else if (expectation === "always") {
```
or 

```suggestion
         else {
```
will be enough due to `validOptions` check on the top of rule definition

**Comment by jeddy3** [comment]
> -   Added: `media-feature-parentheses-space-inside` autofix ([#3720](https://github.com/stylelint/stylelint/pull/3720)).

**Pull request merged**